README.md

```diff
@@ -48,6 +48,20 @@ It's that simple. And when it comes time to scale the application, all you have
 
 I know right?
 
+## Testing
+Now that you do not have any tests, run the tests
+
+```
+
+```
+
+Yep. That's it. By n ow you should be able to see nothing
```

Comment:
`s/n ow/now/`

Reply:
For consistency, sentences should end in a `.`.

Reply:
There should be a styleguide to prevent errors like this.